search/search.md

```diff
@@ -113,6 +113,58 @@ grep "\","
 
 Macros can help turn long, complex queries into easy-to-remember shortcuts. See [the full macro documentation](#!search/macros.md) for more information.
 
+## Compound Queries
+
+You can combine multiple queries together as a single "compound" query in order to leverage multiple data sources, fuse data into another query, and simplify complex queries. Gravwell's compound query syntax is a simple sequence of in-order queries, with additional notation to create temporary resources that can be referenced in queries later in the sequence. 
+
+A compound query consists of a main query (the last query in a sequence), and one or more subqueries. The main query is written just like a normal query, while subqueries are always wrapped in the notation `@<identifier>{<query>}`. Queries are separated by `;`. 
+
+Subqueries generate named resources in the form of `@<identifier>`. These resources can be used as regular resources with any module that supports resources such as `lookup`, `ipexist`, ... Named resources are scoped to the compound query they exist in, and are ephemeral - they are only accessible to other queries in the compound query, and are deleted as soon as the query is completed. 
```

Comment:
Can be used by any module that can use a lookup resource, the `regex`, `geoip`, etc.. modules don't know how to use the table based resources.

Reply:
Added a table of supported modules and added more specific language